GETTING_STARTED.ipynb

```diff
@@ -17,8 +17,29 @@
    "metadata": {},
```

Comment:
<p>Don't have Seaborn available on the add-on end,  but will check if I can just add it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/home-assistant/home-assistant-notebooks/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>